### ISSUE TITLE
Docker vs Systemctl

### ISSUE THREAD

**Issue by osamaramihafez** [SimpleDocUTM/SimpleDoc#68]
our frontend hits https, which implies that we need to use an ssl along with our backend django server. We have the gunicorn command, but accessing the ssl is troublesome. Seems cumbersome to add ssl to backend for production (multiple links I've come across suggest using nginx images... etc). May end up just resorting to sytemctl for serving purposes, which has already been set up on prod server by @shadowshadow725.

**Comment by osamaramihafez** [comment]
Might be a good idea to just use docker for development only

**Comment by shadowshadow725** [comment]
I'm not opposed to using docker but from what we saw yesterday, I don't think docker plays nice with the applications we are running. With systemctl we only need to update the certificate yearly which in my opinion isn't an issue.

**Comment by osamaramihafez** [comment]
I agree, I'm honestly thinking if we toss shibboleth we can forget about docker and just use firebase for log in and registration purposes. It would be more of a public access forum rather than something tied down to one org.
